config/checkstyle_checks.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC
-          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+          "-//Puppy Crawl//DTD Check Configuration 1.4//EN"
+          "http://checkstyle.sourceforge.net/dtds/configuration_1_4.dtd">
```

Comment:
as we do not introduce any new functionality in DTD , lets keep it version.
Please use http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd , it should work.

Please do the same for others.